tflint init

## Describe the bug

The tflint bundled plugins are being [deprecated ](https://github.com/terraform-linters/tflint/pull/1160). A `.tflint.hcl` file is needed to specify which plugins are needed and running `tflint --init` is needed to install the plugins. 

## Expected behavior

Super-linter runs `tflint --init` ahead of running `tflint`.

## Steps to Reproduce

1. Create a `.tflint.hcl` file, specifying at least one plugin.

```hcl
plugin "aws" {
    enabled = true
    version = "0.12.0"
    source  = "github.com/terraform-linters/tflint-ruleset-aws"
}
```

2. Run super-linter action. 

## Logs

```shell
[202](https://github.com/novationtech/aws-service-desk-telegraf-tf/runs/5451120432?check_suite_focus=true#step:4:202)2-03-07 15:55:23 [ERROR]   Found errors in [tflint] linter!
2022-03-07 15:55:23 [ERROR]   Error code: 1. Command output:
------
Failed to initialize plugins. An error occurred:

Error: Plugin `aws` not found. Did you run `tflint --init`?
```

## Additional context

The config file is needed in the current directory. To be useful by super-linter, it is also needed in `.github/linters`. This will require symlink or duplicate files.

--- a/lib/functions/detectFiles.sh
+++ b/lib/functions/detectFiles.sh
@@ -497,4 +497,32 @@ function RunAdditionalInstalls() {
       fi
     fi
   fi
+
+  ####################################
+  # Run installs for TFLINT language #
+  ####################################
+  if [ "${VALIDATE_TERRAFORM_TFLINT}" == "true" ] && [ "${#FILE_ARRAY_TERRAFORM_TFLINT[@]}" -ne 0 ]; then
+    info "Detected TFLint Language files to lint."
+    info "Trying to install the TFLint init inside:[${WORKSPACE_PATH}]"
+    #########################
+    # Run the build command #
+    #########################
+    BUILD_CMD=$(cd "${WORKSPACE_PATH}" || exit 0; tflint --init 2>&1)
+
+    ##############
+    # Error code #
+    ##############
+    ERROR_CODE=$?
+
+    ##############################
+    # Check the shell for errors #
+    ##############################
+    if [ "${ERROR_CODE}" -ne 0 ]; then
+      # Error
+      warn "ERROR! Failed to run:[tflint --init] at location:[${WORKSPACE_PATH}]"
+      warn "BUILD_CMD:[${BUILD_CMD}]"
+    else
+      info "Successfully ran:[tflint --init] in workspace:[${WORKSPACE_PATH}]"
+    fi
+  fi
 }

--- a/lib/functions/detectFiles.sh
+++ b/lib/functions/detectFiles.sh
@@ -507,7 +507,10 @@ function RunAdditionalInstalls() {
     #########################
     # Run the build command #
     #########################
-    BUILD_CMD=$(cd "${WORKSPACE_PATH}" || exit 0; tflint --init 2>&1)
+    BUILD_CMD=$(
+      cd "${WORKSPACE_PATH}" || exit 0
+      tflint --init 2>&1
+    )
 
     ##############
     # Error code #

